assets/scripts/script.js

```diff
@@ -84,5 +111,54 @@ const implex = (function () {
     }
   }
 
+  function onSubmit(e) {
+    e.preventDefault();
+
+    const errorBorder = 'error-border';
+    const hidden = 'hidden';
+
+    const xhr = new XMLHttpRequest();
+    const nameInputField = document.getElementById('contact-form-name-input');
+    const emailInputField = document.getElementById('contact-form-email-input');
+    const commentInputField = document.getElementById('contact-form-comment-input');
+    let nameInputValue = nameInputField.value;
+    let emailInputValue = emailInputField.value;
+    let commentInputValue = commentInputField.value;
+
+    const errorBlock = document.getElementById('contact-form-error-block');
+    const successBlock = document.getElementById('contact-form-success-block');
+
+    const formData = new FormData(contactForm);
+
+    if (!nameInputValue || !emailInputValue || !commentInputValue) {
+      errorBlock.style.display = 'flex';
+
+      if (!nameInputValue) {
+        nameInputField.classList.add(errorBorder);
+      }
+
+      if (!emailInputValue) {
+        emailInputField.classList.add(errorBorder);
+      }
+
+      if (!commentInputValue) {
+        commentInputField.classList.add(errorBorder);
+      }
+    } else {
+      nameInputField.style.borderColor = '#7A7878';
```

Comment:
You should have it in scss as default value, that will be changed by adding `error-border` class

Reply:
And don't forget to remove `errorBorder ` class somewhere